### PR TITLE
#81: Members aren't nameless account holders

### PR DESCRIPTION
**Pull request by tbfleming**

### CODE REVIEW

**Comment by brandonfancher** [comment]
This LGTM, but I'll leave it to @sparkplug0025 or @swatanabe to approve. Also, once we merge and redeploy, we'll need to re-init the community, right?

**Comment by tbfleming** [comment]
yes

File: contracts/eden/include/members.hpp [hunk 1]
```diff
@@ -69,7 +70,7 @@ namespace eden
       void check_pending_member(eosio::name account);
       void deposit(eosio::name account, const eosio::asset& quantity);
       void set_nft(eosio::name account, int32_t nft_template_id);
-      void set_active(eosio::name account);
+      void set_active(eosio::name account, std::string name);
```
Comment (swatanabe) [inline contracts/eden/include/members.hpp:73]: `const&`

**Review by swatanabe — APPROVED** [review]

File: contracts/eden/include/members.hpp (hunk 1 above)
Comment (tbfleming) [reply contracts/eden/include/members.hpp]: fixed

**Review by swatanabe — APPROVED** [review]

**Comment by sparkplug0025** [comment]
you forgot to update abi! pls let's try to keep it updated 😛

**Pull request merged**